src/utils/csv_reader.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"encoding/csv"
+	"log"
+	"os"
+)
+
+func CsvReader(path string) ([][]string, error) {
```

Comment:
It is a good practice to put a comment about exported interface, structures, and methods, whenever we declare something with Capitalize it is exported for other packages to be able to use it.